components/Form.js

```diff
@@ -1,69 +1,92 @@
 import { styled } from "styled-components";
 import PrimaryLink from "./PrimaryLink";
+import { uid } from "uid";
+const slugify = require("slugify");
```

Comment:
please use the standart way to import libraries with `import <name> from "package-name"`